Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -303,14 +307,14 @@ def github_status_for_queue(self, change, queue):
             elif re.search(r'Hash .* on PR .* is outdated', build.state_string):
                 hover_over_text += ' Commit was outdated when EWS attempted to process it.'
         elif build.result == Buildbot.RETRY:
-            hover_over_text = 'Build is being retried'
+            hover_over_text = 'Build is being retried. Recent messages:' + self._steps_messages(build)
```

Comment:
So, my issue with this is our insertion of it into the PR description. Because we already know that hitting the character limit isn't entirely theoretical for certain large changes. I'm not going to block on this concern, but the more feature we add here, the more possible issue's we're going to encounter.

Reply:
Yeah, that make sense.